docker/bootstrap.sh

```diff
@@ -37,6 +37,18 @@ else
 
     mkdir -p /sytest
     tar -C /sytest --strip-components=1 -xf sytest.tar.gz
+
+    if [ -n "$PLUGIN" ]; then
+        mkdir /sytest/plugins
+        echo "--- Downloading plugins for sytest"
+        IFS=':'; for plugin in $PLUGIN; do
+            wget -q https://github.com/$plugin/archive/master.tar.gz -O plugin.tar.gz || {
+                echo "Failed to download plugin: $plugin"
```

Comment:
we write this (to stdout rather than stderr) and then carry on with mkdir and tar anyway? seems like we should exit instead? or just let `set -e` handle it?

Reply:
Thanks for finding this. Yeah we should exit there with sending the msg to stderr.